packages/server/graphql/public/mutations/startRecurrence.ts

```diff
@@ -0,0 +1,73 @@
+import ms from 'ms'
+import MeetingSeriesId from 'parabol-client/shared/gqlIds/MeetingSeriesId'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {RRule} from 'rrule'
+import getRethink from '../../../database/rethinkDriver'
+import {insertMeetingSeries as insertMeetingSeriesQuery} from '../../../postgres/queries/insertMeetingSeries'
+import {getUserId} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import {MutationResolvers} from '../resolverTypes'
+
+const startRecurrence: MutationResolvers['startRecurrence'] = async (
+  _source,
+  {meetingId},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const r = await getRethink()
+  const viewerId = getUserId(authToken)
+  const now = new Date()
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+
+  // VALIDATION
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  if (!meeting) {
+    return standardError(new Error('Meeting not found'), {userId: viewerId})
+  }
+
+  if (meeting.meetingType !== 'teamPrompt') {
+    return standardError(new Error('Meeting is not a team prompt meeting'), {userId: viewerId})
+  }
+
+  const {teamId, meetingSeriesId} = meeting
```

Comment:
-1 check viewer permissions (either they have a MeetingMember, or they are a teamMember, up to you)

Reply:
Nice catch! Checked whether the user is a team member